website/content/docs/configure.mdx

```diff
@@ -119,7 +119,7 @@ each can be found below:
   new versions of Packer. If you want to disable this for security or privacy
   reasons, you can set this environment variable to `1`.
 
-- TMPDIR (Unix) / TMP, TEMP, USERPROFILE (Windows) - This specifies the
+- `TMPDIR` (Unix) / `TMP`, `TEMP`, `USERPROFILE` (Windows) - This specifies the
      directory for temporary files (defaulting to /tmp on Linux/Unix and
      %USERPROFILE%\AppData\Local\Temp on Windows Vista and later). Customizing
```

Comment:
If we're updating this documentation I would suggest fencing the `/tmp` and `%USERPROFILE` for better readability